servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/PipelinedLBHttpConnectionFactory.java

```diff
@@ -40,12 +40,13 @@
             final ReadOnlyHttpClientConfig config, final HttpExecutionContext executionContext,
             @Nullable final StreamingHttpConnectionFilterFactory connectionFilterFunction,
             final StreamingHttpRequestResponseFactory reqRespFactory,
-            final HttpExecutionStrategy chainStrategy,
+            final ExecutionStrategy connectStrategy,
             final ConnectionFactoryFilter<ResolvedAddress, FilterableStreamingHttpConnection> connectionFactoryFilter,
             final Function<FilterableStreamingHttpConnection,
                     FilterableStreamingHttpLoadBalancedConnection> protocolBinding) {
-        super(config, executionContext, connectionFilterFunction, version -> reqRespFactory, chainStrategy,
-                connectionFactoryFilter, protocolBinding);
+        super(config, executionContext, version -> reqRespFactory,
+                connectStrategy, connectionFactoryFilter, connectionFilterFunction,
```

Comment:
nit: we generally try to fill the 120 chars up on a line unless it impacts readability. the additional newline here doesn't seem to help readability.